docs/designers-developers/developers/slotfills/README.md

```diff
@@ -0,0 +1,109 @@
+# SlotFills
```

Comment:
```suggestion
# SlotFills Reference
```

This title is used both in the handbook page and in the sidebar. I'd rather use the same pattern we use in previous sections (Block API Reference, Filter Reference).